math/mathcore/test/testVectorizedTMath.cxx

```diff
@@ -0,0 +1,84 @@
+#include <random>
+#include "VectorizedTMath.h"
+#include <gtest/gtest.h>
+
+#if defined(R__HAS_VC)
```

Comment:
I think you probably need `R__HAS_VECCORE` here too.

Reply:
Ah yes...I made the changes `#if defined(R__HAS_VECCORE) && defined(R__HAS_VC)` locally.
Trying to figure out the issue with `NormQuantile`.